src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-linux`
Test-Connection.Ping.Default parameter set is 'Ping'
```powershell
Expected the value to have type [System.Net.NetworkInformation.PingOptions] or any of its subtypes, but got $null with type $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 45
45:             $result.Options | Should -BeOfType "System.Net.NetworkInformation.PingOptions"
```
Test-Connection.MTUSizeDetect.MTUSizeDetect works
```powershell
Expected the actual value to be greater than 0, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 221
221:             $result.MtuSize | Should -BeGreaterThan 0
```